src/Benchmarks/Program.cs

```diff
@@ -186,5 +187,21 @@ private static void Listen(KestrelServerOptions options, IConfigurationRoot conf
                 }
             });
         }
+
+        private static IPEndPoint CreateIPEndPoint(Uri uri)
+        {
+            IPAddress ip;
+
+            if (string.Equals(uri.Host, "localhost", StringComparison.OrdinalIgnoreCase))
+            {
+                ip = IPAddress.Loopback;
```

Comment:
Technically I think we should bind to both `IPAddress.Loopback` and `IPAddress.IPv6Loopback`, like we do at https://github.com/aspnet/KestrelHttpServer/blob/2351c1b558fc6f4312af262aa866fd2c5164761f/src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs#L230.